checker/src/functions/check-subgraph/handler.ts

```diff
@@ -0,0 +1,94 @@
+import type { Request, Response } from 'express'
+import axios from 'axios'
+import pRetry from 'p-retry'
+
+type ConfigType = {
+  CHECKER_RUNTIME_SUBGRAPH_URL: string
+  CHECKER_RUNTIME_ETHEREUM_PROVIDER_URL: string
+  CHECKER_RUNTIMETHRESHOLD_BLOCKS_LAGGING: string
+}
+
+// load from runtime environment variables
+const getConfig = (): ConfigType => {
+  return {
+    CHECKER_RUNTIME_SUBGRAPH_URL: process.env.CHECKER_RUNTIME_SUBGRAPH_URL!,
+    CHECKER_RUNTIME_ETHEREUM_PROVIDER_URL: process.env
```

Comment:
This breaks @typescript-eslint/no-non-null-assertion when I read the code in vscode. I think it's picking up the rules from subgraph main eslint config ..